src/riak_cs_console.erl

```diff
@@ -33,6 +34,11 @@
 %%% Public API
 %%%===================================================================
 
+status([]) ->
+    Stats = lists:sort(riak_cs_stats:get_stats()),
```

Comment:
Do we need sort here? This sorting orders item as `100` -> `95` -> `99`:

```
block_get_retry_time_100 : 0
block_get_retry_time_95 : 0
block_get_retry_time_99 : 0
block_get_retry_time_mean : 0
block_get_retry_time_median : 0
```

On the other hand, JSON output is, for me, more human friendly:

```
    "block_get_retry_time_mean": 0,
    "block_get_retry_time_median": 0,
    "block_get_retry_time_95": 0,
    "block_get_retry_time_99": 0,
    "block_get_retry_time_100": 0,
```